metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license          'Apache-2.0'
 description      'Sets up SSSD for LDAP on Ubuntu and RHEL systems'
-version          '5.1.2'
+version          '5.1.3'
```

Comment:
Can you remove the version change please as it gets bumped automatically when the cookbook is released